testregistry.textproto

```diff
@@ -793,6 +793,13 @@ test: {
   readme: "https://github.com/openconfig/featureprofiles/blob/main/feature/aft/aft_summary/otg_tests/route_summary_counters_test/README.md"
   exec: " "
 }
+test: {
+  id: "RT-4.11"
+  description: " Scale AFTs Route Summary"
+  readme: "https://github.com/openconfig/featureprofiles/blob/main/feature/aft/aft_summary/otg_tests/scale_aft_summary/README.md"
+  exec: " "
```

Comment:
This would be a link to the route_test you wrote?

Reply:
in practice, we do not use the exec field, it is ok to remove it.